docs/api.rst

```diff
@@ -12,7 +12,7 @@ state_space
 
     StateSpace
     StateSpace.get_attribute_from_period
-    StateSpace.update_systematic_rewards
```

Comment:
I felt the new name is more descriptive about what is going on, but you are right, it is less general because it only refers to wage and nonpec components. But what is the purpose of systematic?

Reply:
Systematic means that the shocks are not included yet. I think this is a quite common terminology.